src/css/main.less

```diff
@@ -870,6 +870,10 @@ fieldset[disabled] .btn-info.active {
   }
 }
 
+.pane-header {
+  line-height: 30px;
```

Comment:
We should use a variable on some point, because I am using the same value (30px) in my dialogs-PR.
Maybe we can derive this from bootstrap directly..